pkg/gedis/types/directory/tfgrid_node_2.go

```diff
@@ -9,6 +9,7 @@ import (
 //TfgridNode2 jsx schema
 type TfgridNode2 struct {
 	NodeID            string                    `json:"node_id"`
+	NodeIDv1          string                    `json:"node_id_v1"`
```

Comment:
Once we have this, we will have it forever. Which means, even nodes that never run v1 software, will still have this set. with no real use.

May be, the V1 version should be sent as a separate argument to the register-node method. And bcdb can use it to find a v1 node (and do whatever linking needed), otherwise ignore it. But then the node data will be clean, with no useless garbage.

What do u think?

Reply:
Well since there is no way to know from the node if it was running v1 before or not, we will anyhow always send it. So that seems fine for now. Also this change match the one in the model in BCDB.

Reply:
I know that we will always send it, but we don't have to keep it on the node object, is what I am trying to say